src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -185,15 +190,16 @@ override Microsoft.CodeAnalysis.CSharp.Syntax.VariableDeconstructionDeclaratorSy
 override Microsoft.CodeAnalysis.CSharp.Syntax.WhenClauseSyntax.Accept(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor visitor) -> void
 override Microsoft.CodeAnalysis.CSharp.Syntax.WhenClauseSyntax.Accept<TResult>(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor<TResult> visitor) -> TResult
 static Microsoft.CodeAnalysis.CSharp.CSharpExtensions.GetDeclaredSymbol(this Microsoft.CodeAnalysis.SemanticModel semanticModel, Microsoft.CodeAnalysis.CSharp.Syntax.DeclarationPatternSyntax declarationSyntax, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> Microsoft.CodeAnalysis.ISymbol
+static Microsoft.CodeAnalysis.CSharp.SyntaxExtensions.Identifier(this Microsoft.CodeAnalysis.CSharp.Syntax.DeclarationExpressionSyntax self) -> Microsoft.CodeAnalysis.SyntaxToken
```

Comment:
If I recall correctly, I thought we had left the discussion thinking we'd keep those two helpers internal for now.